plugins/kaniko.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kaniko
+spec:
+  version: "v1.1.0"
+  homepage: https://github.com/kvaps/kubectl-kaniko
+  shortDescription: Kaniko wrapper for kubectl
+  description: |
+    Build your Dockerfiles directly in your Kubernetes cluster.
```

Comment:
Please state that you run a kaniko executor under the hood to build the images.

Also for those not familiar with kaniko you should briefly explain where the built image ends up.

Reply:
description updated!